docs/explanations/audit-scanner/audit-scanner.md

```diff
@@ -76,7 +74,7 @@ See the policy authors [documentation](../../tutorials/writing-policies/index.md
 
 The audit scanner in Kubewarden requires specific Role Based Access Control (RBAC) configurations to be able to scan Kubernetes resources and save the results.
 A correct default Service Account with those permissions is created during the installation.
-The user can create and configure their own ServiceAccount to fine tune access to resources.
+The user can create and configure their own ServiceAccount to fine-tune access to resources.
```

Comment:
Maybe?

```suggestion
The user can create their own ServiceAccount to configure access to resources.
```

Idiomatic English like 'fine-tune' is great, but it does rely on shared understanding that may not always be in place with an international audience.

In this case the audience need to understand it's to do with getting musical instruments to sound correct, or perhaps it refers to making sure that an engine is operating optimally, or some of the phrases many other uses. But 'fine-tune' __is__ widely used, so .....